kernel/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/constant/RegistrationCenterMachineErrorCode.java

```diff
@@ -16,8 +16,10 @@
 	REGISTRATION_CENTER_MACHINE_DELETE_EXCEPTION("KER-MSD-106",
 			"Error occurred while deleting a mapping of Machine and Center"),
 	REGISTRATION_CENTER_MACHINE_FETCH_EXCEPTION("KER-MSD-601", "Error occurred while fetching Center Machine details"),
-	REGISTRATION_CENTER_MACHINE_ZONE_INVALID("KER-MSD-411",
-			"Admin not authorized to map/un-map this Registration Center or Machine"),
+	REGISTRATION_CENTER_ZONE_INVALID("KER-MSD-411",
+			"Admin not authorized to map/un-map this Registration Center"),
+	REGISTRATION_MACHINE_ZONE_INVALID("KER-MSD-411",

```

Comment:
Will these both cases have same error code?

Reply:
No , it will not . I have fixed the mistake by changing the error code to KER-MSD-412 as per the user story.